src/Sentry/Internal/Hub.cs

```diff
@@ -659,9 +659,7 @@ public void Dispose()
 
         try
         {
-            Metrics.FlushAsync().ContinueWith(_ =>
-                CurrentClient.FlushAsync(_options.ShutdownTimeout).ConfigureAwait(false).GetAwaiter().GetResult()
-            ).ConfigureAwait(false).GetAwaiter().GetResult();
+            CurrentClient.FlushAsync(_options.ShutdownTimeout).ConfigureAwait(false).GetAwaiter().GetResult();
```

Comment:
Are we marking it as obsolete or removing the functionality in this PR? This change is removing metrics functionality right?

Reply:
Ah, I see in the PR description:
> While marking the API as obsolete I also removed the shutdown flushing. I think it's fairly low risk and it fixes the deadlocking situation in https://github.com/getsentry/sentry-unity/issues/1779